action-helm-tools/README.md

```diff
@@ -44,25 +42,27 @@ steps:
 ## Required Environment variables
 
 ```yaml
-ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }} # ARTIFACTORY_USERNAME (Artifactory username) must be set in GitHub Repo secrets
-ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }} # ARTIFACTORY_PASSWORD (Artifactory api key) must be set in GitHub Repo secrets
+QLIK_DOCKER_DEV_REGISTRY:
+QLIK_DOCKER_DEV_USERNAME:
+QLIK_DOCKER_DEV_PASSWORD:
+QLIK_HELM_DEV_REGISTRY:
+QLIK_HELM_DEV_USERNAME:
+QLIK_HELM_DEV_PASSWORD:
```

Comment:
Ideally, the code in qlik-oss should not have any references to qlik, but in practice it is probably too tedious to change the names of those variables to something more general.